### PR TITLE
Create feature-lifecycle-and-deprecation

### PR DESCRIPTION
**Pull request by qnetter**
Policy proposal for community preview.

### CODE REVIEW

**Review by Vad1mo — APPROVED** [review]
good idea

**Review by OrlinVasilev — APPROVED** [review]
lgtm

File: proposals/feature-lifecycle-and-deprecation [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Deprecation Process
+
+Any contributor may introduce a request to deprecate a feature or an option of a feature by opening a feature request issue in the goharbor/harbor GitHub project. The issue should describe why the feature is no longer needed or has become detrimental to Harbor, as well as whether and how it has been superseded. The submitter should give as much detail as possible.
+
+Once the issue is filed, a discussion period begins which ends at the beginning of the second community meeting after the opening of the issue, and is held at the community meeting and on the issue itself; the person who opens the issue or a maintainer should add that date and time in a comment in the issue as soon after the issue is opened as possible.
+
+The feature will be deprecated by a supermajority vote of 50% plus one of the project maintainers at the time of the vote tallying, which is 72 hours after the end of the community meeting that is the end of the comment period. (Maintainers are permitted to vote in advance of the deadline, but should hold their votes until as close as possible to hear all possible discussion.) Votes will be tallied in comments on the issue.
```
[the remark this answers is not in the document]
Comment (qnetter) [reply proposals/feature-lifecycle-and-deprecation:28]: > How we are going to track that timing?

We can track in deprecation issue comments and meeting minutes.  Issue comments should be authoritative.

**Comment by OrlinVasilev** [comment]
looks great to me
BTW @qnetter I have enabled DCO(signing) from web interface for that repository and website can you give it a try ?

Thanks

**Comment by qnetter** [comment]
So please note that the voting period is already underway, and people should comment on this issue with +1 (and any comments) to approve, 0 to neither approve nor disapprove, or -1 to disapprove; non-maintainers should add "nb" for non-binding (though of course we know who the maintainers are). Votes will be tallied at the next community meeting. @OrlinVasilev please comment with that date/time below. @goharbor/all-maintainers please vote.

**Comment by qnetter** [comment]
Completed.

**Pull request merged**